Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -129,11 +140,17 @@ public class MongoDatabase {
         self._database = nil
     }
 
-    /// Drops this database.
-    /// - Throws:
-    ///   - `ServerError.commandError` if an error occurs that prevents the command from executing.
-    public func drop(session: ClientSession? = nil) throws {
-        let operation = DropDatabaseOperation(database: self, session: session)
+   /**
+    *   Drops this database.
+    * - Parameters:
+    *   - session: An optional `ClientSession` to use for this command
+    *   - options: Optional `DropDatabaseOptions` to use when executing this command
+    *
+    * - Throws:
+    * - `ServerError.commandError` if an error occurs that prevents the command from executing.
+    */
+    public func drop(session: ClientSession? = nil, options: DropDatabaseOptions? = nil) throws {
```

Comment:
The order of these parameters should be switched so that session is the last one to correspond with our other methods.